src/tests/mocks/factories.ts

```diff
@@ -177,14 +190,28 @@ const translationUiFactory = () => ({
   'components.WidgetPropertiesForm.title': 'Nom du champ',
 });
 
+function transformErrors(errors: AjvError[]): AjvError[] {
+  return errors.map((error) => {
+    if (error.name === 'minimum') {
+      error.message = `The value must be ${error.params.comparison} to ${error.params.limit}`;
+    }
+    if (error.name === 'maximum') {
+      error.message = 'Your value exceeds the maximum';
+    }
+    return error;
+  });
+}
```

Comment:
I wonder if we really need to put this function here. I think, it could be moved in its related test suite ?

Reply:
Indeed, i put it before renaming as factories, my bad